### PR TITLE
munin-libvirtpy now in python3

### PR DESCRIPTION
**Pull request by AloisKlingler**
works on debian bullseye.
flake8 compatiblity ckecked.
I am not sure with the modified header, just copy&pasted - please feel free to adapt it to be compatible and as wanted - thanks. :-)

### CODE REVIEW

File: plugins/libvirt/munin-libvirtpy [hunk 1]
```diff
@@ -1,57 +1,75 @@
...
+if conn is None:
+    print('Failed to open connection to the hypervisor')
     sys.exit(1)
 
 try:
     (model, memory, cpus, mhz, nodes, socket, cores, threads) = conn.getInfo()
-except:
-    print 'getInfo failed'
+except BaseException as error:
+    print('getInfo failed: {}'.format(error))
```
Comment (sumpfralle) [inline plugins/libvirt/munin-libvirtpy:49]: Please write error messages to `stderr` (`file=sys.stderr`).

File: plugins/libvirt/munin-libvirtpy [hunk 2]
```diff
@@ -1,57 +1,75 @@
...
 
-#print
-#print "KVM running on %d %s %d mhz CPUs w/ %d MB RAM." % (cpus, model, mhz, memory)
-#print
-
 ids = conn.listDomainsID()
-if ids == None or len(ids) == 0:
-    print 'No running domains found.'
+if ids is None or len(ids) == 0:
+    print('No running domains found.')
```
Comment (sumpfralle) [inline plugins/libvirt/munin-libvirtpy:54]: `stderr`, please.

**Review by sumpfralle — COMMENTED** [review]
Thank you for tackling the python3 migration for this plugin!

Below I added two minor comments for desirable changes (unrelated to the python3 migration).
I would be happy, if you could fix these too, while you are  involved with the plugin.
But I would also just merge your changes, as they are now (since they are perfectly fine for a python3 migration).
Just tell me, which path you choose ...

Thanks!

File: plugins/libvirt/munin-libvirtpy (hunk 1 above)
Comment (AloisKlingler) [reply plugins/libvirt/munin-libvirtpy]: done.
as I do all the changes in web, without having git installed, I cannot squash them as you suggested (or at least I cannot find a way in web to do it). please can you do again this? Thanks!

File: plugins/libvirt/munin-libvirtpy (hunk 2 above)
Comment (AloisKlingler) [reply plugins/libvirt/munin-libvirtpy]: done.
as I do all the changes in web, without having git installed, I cannot squash them as you suggested (or at least I cannot find a way in web to do it). please can you do again this? Thanks!

**Comment by sumpfralle** [comment]
Thank you for taking care for this plugin!

**Pull request merged**